contracts/deploy-history/arbitrum-2022-03-02T18:56:10.776Z.json

```diff
@@ -0,0 +1,8 @@
+{
+  "deployer": "0x2217432e5E0e2D77576461662d267bd293820811",
+  "contracts": {
+    "Umbra": "0x2C943761FEA42Eaca0AfE6064A6daB935fB5b18a",
```

Comment:
Seems there's a few deploy files where the contracts were deployed to the wrong address, so we should probably delete those to avoid confusion in case any users poke around. Guessing you were doing a test deployment, but I don't think we need to care about the fact these exist and don't need to keep the files. IMO we should only keep the real deploy files that we care about

And actually, since every "real" file is the same, we can simplify this whole `deploy-history` folder into a single `deploys.json` file that maps chain ID to deploy addresses. But we don't need to do that in this PR

Reply:
Given we timestamp the deployments and keep the `-latest` version up to date automatically, I don't see much harm in keeping around the records of the test deployments. They could even come in handy for testing or something in the future and this gives us an easy way to know where they are.

> And actually, since every "real" file is the same, we can simplify this whole deploy-history folder into a single deploys.json file that maps chain ID to deploy addresses. But we don't need to do that in this PR

Are you looking for something you can import into the frontend codebase or something along those lines? If so, I'm not opposed to creating it, but I think it could/should be separate from the deployment history. We could put a `json` file in the root of the contract dir, for example.

Reply:
>  I don't see much harm in keeping around the records of the test deployments. They could even come in handy for testing or something in the future and this gives us an easy way to know where they are.

`-latest` does not imply production deploy though, just that it was the most recent deploy, so it still just feels confusing to have useless non-production deploys in this folder — what's the canonical way to find the production deploy? If we keep this, it seems the answer would be the FAQ or umbra-js code, but it feels like that should be somewhere in the contracts folder

> Are you looking for something you can import into the frontend codebase or something along those lines? If so, I'm not opposed to creating it, but I think it could/should be separate from the deployment history. We could put a json file in the root of the contract dir, for example.

Nope, just looking to make things cleaner and easier to read. This folder looks extremely cluttered given that there's just a handful of total addresses (across all networks) that actually matter
<img width="1144" alt="image" src="https://user-images.githubusercontent.com/17163988/157260813-475bb654-009b-4ec2-b74b-91cca4c3eb24.png">

Reply:
Tracking in https://github.com/ScopeLift/umbra-protocol/issues/306 to unblock this PR